python/cuml/test/test_random_forest.py

```diff
@@ -193,14 +186,19 @@ def test_rf_classification_default(datatype, column_info, nrows):
     # compare the accuracy of the two models
     # github issue 1306: had to increase margin to avoid random CI fails
     # assert cu_acc >= (sk_acc - 0.07)
-    assert cu_acc >= (sk_acc - 0.2)
+    try:
+        assert fil_acc >= (cu_acc - 0.2)
+        assert fil_acc >= (sk_acc - 0.7)
```

Comment:
Forgot to add this in the review, this margins need to be adjusted, no? I changed it to 0.2 just to not block the numba 0.46 PR, but if the test (or algo) works correctly, this should be smaller, no? (I think this might be a typo and you meant 0.02 and 0.07)

Reply:
Yes, it should be 0.02 and 0.07